app.py

```diff
@@ -56,6 +93,17 @@ def scrape():
     except Exception as e:
         print('Scraping error:', str(e))
         return jsonify({'error': 'An error occurred while scraping'}), 500
+    
+@app.route('/scrape-amenities')
+def scrapeAmenities():
```

Comment:
as a new task make this more dynamic. in the url  params we will take input of airbnb property url instead of it being static.

also. save the reviews data and amenities to a database like mongodb